static/src/sass/front/form/_layout_foundation.scss

```diff
@@ -0,0 +1,155 @@
+.noviusos_form {
+
+    /* The form layout */
+    .form-errors {
+        color: darkred;
+        margin: 10px 0;
+        text-align: left;
+    }
+
+    .column, .columns {
+        padding-right: 10px;
+    }
+
+    label[for] {
+        cursor: pointer;
+    }
+    //
+    //input {
+    //
+    //    &[type="text"],
+    //    &[type="password"],
+    //    &[type="date"],
+    //    &[type="datetime"],
+    //    &[type="email"],
+    //    &[type="number"],
+    //    &[type="search"],
+    //    &[type="tel"],
+    //    &[type="time"],
+    //    &[type="url"] {
+    //        background-color: white;
+    //        font-family: inherit;
+    //        border: 1px solid #cccccc;
+    //        -webkit-border-radius: 2px;
+    //        -moz-border-radius: 2px;
+    //        -ms-border-radius: 2px;
+    //        -o-border-radius: 2px;
+    //        border-radius: 2px;
+    //        -webkit-box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
+    //        -moz-box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
+    //        box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
+    //        color: rgba(0, 0, 0, 0.75);
+    //        -webkit-transition: all 0.15s linear;
+    //        -moz-transition: all 0.15s linear;
+    //        -o-transition: all 0.15s linear;
+    //        transition: all 0.15s linear;
+    //
+    //        &:focus {
+    //            background: #fafafa;
+    //            outline: none !important;
+    //            border-color: #b3b3b3;
+    //        }
+    //
+    //        &[disabled] {
+    //            background-color: #ddd;
+    //        }
+    //    }
+    //}
+
+    input,
+    select,
+    textarea {
+
+        /* Don't move the :focus pseudo selector, or it won't work in Chrome */
+        &.form-ui-invalid,
+        &:focus.form-ui-invalid {
+            border-color: #cc0000;
+        }
+    }
+
+    .label-position-top,
+    .label-position-placeholder {
+
+        input[type="text"],
```

Comment:
that's a variable's job http://www.sassmeister.com/gist/70399284101b9c506d757cf143094c88